src/scales/scale.linearbase.js

```diff
@@ -15,6 +15,8 @@ function generateTicks(generationOptions, dataRange) {
 	// "nice number" algorithm. See https://stackoverflow.com/questions/8506881/nice-label-algorithm-for-charts-with-minimum-ticks
 	// for details.
 
+	// Minimum spacing between ticks
```

Comment:
this comment might not be needed either. If we want to capture the "between ticks" part we could rename the variable `MIN_TICK_SPACING` though I'm not sure that's necessary